crates/uv/tests/pip_compile.rs

```diff
@@ -5224,7 +5224,7 @@ fn unsupported_scheme() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    error: Unsupported scheme `bzr+https` on URL: bzr+https://example.com/anyio (Bazaar is not supported)
+    error: Unsupported URL prefix `bzr` in URL: `bzr+https://example.com/anyio`
```

Comment:
I feel like this error message got worse, can we restore it?

Reply:
At least the part at the end.

Reply:
https://github.com/astral-sh/uv/pull/3403

I'd also like to track the origin of the invalid url, but that's a larger task.